xilem/src/view/label.rs

```diff
@@ -21,8 +21,9 @@ pub fn label(label: impl Into<ArcStr>) -> Label {
 }
 
 #[must_use = "View values do nothing unless provided to Xilem."]
+#[derive(PartialEq)]
 pub struct Label {
-    label: ArcStr,
+    pub(in crate::view) label: ArcStr,
 
     // Public for variable_label as a semi-interims state.
```

Comment:
We should add button to this comment and move it up (probably also worth fixing my typo of `interims` when I meant `interim`)